plugins/vue-chart.ts

```diff
@@ -33,79 +33,49 @@ const createCustomChart = () => {
     chart.update()
   }
 
-  Vue.component<ChartVCData, ChartVCMethod, ChartVCComputed, ChartVCProps>(
-    'doughnut-chart',
-    {
-      extends: Doughnut,
-      mixins: [reactiveProp],
-      props: {
-        displayLegends: {
-          type: Array,
-          default: () => null
-        },
-        options: {
-          type: Object as PropType<ChartOptions>,
-          default: () => {}
-        }
+  const generalChart = Vue.component<
+    ChartVCData,
+    ChartVCMethod,
+    ChartVCComputed,
+    ChartVCProps
+  >('general-chart', {
+    mixins: [reactiveProp],
+    props: {
+      displayLegends: {
+        type: Array,
+        default: () => null
       },
-      watch: {
-        displayLegends: watchDisplayLegends
-      },
-      mounted(): void {
-        this.renderChart(this.chartData, this.options)
+      options: {
+        type: Object as PropType<ChartOptions>,
+        default: () => {}
       }
+    },
+    watch: {
+      displayLegends: watchDisplayLegends
+    },
+    mounted() {
+      this.renderChart(this.chartData, this.options)
```

Comment:
```suggestion
      setTimeout(() => {
        this.renderChart(this.chartData, this.options)
      })
```
こうしないと横スクロールを用いているグラフが縮こまってしまいます。
![image](https://user-images.githubusercontent.com/34832037/83969334-648dc080-a90a-11ea-9279-e2bb55f3bf14.png)